sei/assetlist.json

```diff
@@ -23,6 +23,25 @@
                 "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sei/images/sei.svg"
             },
             "coingecko_id": "sei-network"
+        },
+
+        {
+            "description": "OIN Token ($OIN) is a pioneering digital asset developed on the $SEI Blockchain. Representing more than just a cryptocurrency, $OIN is a symbol of the future of decentralized finance, designed with the vision of reshaping the crypto landscape.",
+            "denom_units": [
+                {
+                    "denom": "factory/sei12q0zv3c4cd9jkupn0krazdycc5ftw9wzt9vmhu/OIN",
+                    "exponent": 6
+                }
+            ],
+            "base": "OIN",
+            "name": "OIN",
+            "display": "oin",
```

Comment:
this demon isn't defined. You can add another denom_unit above called 'oin' with exponent 6.

Reply:
Done too